dev-haskell/tagsoup-megaparsec/tagsoup-megaparsec-0.2.0.0.ebuild

```diff
@@ -15,7 +15,9 @@ SRC_URI="mirror://hackage/packages/archive/${PN}/${PV}/${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0/${PV}"
-KEYWORDS="~amd64 ~x86"
+# incompatible with >=dev-haskell/megaparsec-6.0, no repo activity since
+# August 2016.
+# KEYWORDS="~amd64 ~x86"
```

Comment:
Usually it's better to use real masking via `profiles/package.mask` file. That way users with a package installed get a message you can attach to masked entry. Like the hints why package is masked.

Reply:
Good to know. Cheers, I can push an update if you aren't already working on it.